README.md

```diff
@@ -1,20 +1,29 @@
 # [Measurement Lab](http://www.measurementlab.net/) Source Code
 
-This is the source code of the Measurement Lab website built using [Jekyll](http://jekyllrb.com) and utilizing [GitHub Pages](https://pages.github.com/) to publish and host the site.
+This is the source code of the Measurement Lab website built using
```

Comment:
Reverted line reflowing in readme in https://github.com/shredtechular/m-lab.github.io/commit/1bd89ca17e269f20c670ec48ac9ad4090416561a